packages/checkout/sdk/src/types/smartCheckout.ts

```diff
@@ -198,8 +198,8 @@ export type ERC20BuyToken = {
   type: ItemType.ERC20;
   /** The amount of native token. */
   amount: string;
-  /** The contract address of the ERC20. */
-  contractAddress: string;
+  /** The token address of the ERC20. */
+  tokenAddress: string;
```

Comment:
just reminder if we are gonna make this change to update the docs https://docs.immutable.com/docs/zkEVM/products/checkout/sdk/smart-checkout/smart-checkout-sell

Reply:
I've updated references here: https://github.com/immutable/docs/pull/748